make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
This made me cringe and then fart and then I moaned "autoexec" for some inexplicable reason.

Reply:
I'm nuking this file as well. Sphinx cruft.